apps/picsa-tools/crop-probability-tool/src/app/components/crop-probability-table/crop-probability-table.component.html

```diff
@@ -1,8 +1,17 @@
-<div ngClass="table-filter">
+<!-- <div ngClass="table-filter">
   <mat-form-field>
     <mat-label>Filter</mat-label>
     <input matInput (keyup)="applyFilter($event)" placeholder="Search Crops" #input />
   </mat-form-field>
+</div> -->
+<div ngClass="filter-icons">
+  <div (click)="filterbyImage('maize')"><mat-icon svgIcon="picsa_crop_maize"></mat-icon><div>Maize</div></div>
```

Comment:
nit(non-blocking) - it might be better to refactor this to an `ngFor` loop, with a list of crops in the component .ts file. If stored as a json object you could also track the label and svgIcon name